examples/django/README.md

```diff
@@ -0,0 +1,87 @@
+### Requirements
+This example assumes that you have `Django` and `pwnedapi` installed and the Django project contains app named `django_pwnedapi`.
```

Comment:
@nikoheikkila Isn't it will be very toolchain specific? People use all kinds of things, `pipenv`, `venv`, `virtualenv`, `pyenv`, etc. Also, I don't think anyone installs dependencies globally with all those "use virtual envs" reminders everywhere. I thought it's a good idea to focus on integration in the Django integration example.

What kind of change you want here? Another section about installation starting with `virtualenv` and ending with adding Django app?

Reply:
Alright, I added Installation section with basically a tutorial on how to install everything from scratch, see 0ac4a81

Reply:
I was thinking something like this would be sufficient:

> In your Django project run command `pipenv install pwnedapi` or `pip install pwnedapi` depending on what you use.

Current documentation is good too. Let's have it.